app/controllers/api/v1/testimonials_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::TestimonialsController < ApplicationController
```

Comment:
I think you could delete this and keep the rest or delete the rest and keep this, because I think it's the same